notebooks/dataset_tutorial/README.md

```diff
@@ -1,21 +1,62 @@
+Have a look at our white paper for in depth description of the data. [https://link_to_arxiv_submission](https://gin.g-node.org/cajal/Sensorium2022)
 
-- **data:** includes the variables that were recorded during the experiment. The experimental variables are saved as a collection of numpy array where each numpy array contains the value of that variable at a specific trial. Note that the name of the files does not contain any information about the order or time at which the trials took place.
-    - **behavior:** behavioral variables include pupil dilation, running speed, and the derivative of the running speed. The directory contains NumPy arrays where each single `X.npy` contains the behavioral variables (in the same order that was mentioned earlier) for trial `X`.
-    - **images:** This directory contains NumPy arrays where each single `X.npy` contains the image that was shown to the subject in trial `X`.
-    - **responses:** This directory contains NumPy arrays where each single `X.npy` contains the Calcium responses recorded from the subject in trial `X`.
+We provide the datasets in the .zip format. Unzipping them will create two folders **data** and **meta**.
+
+- **data:** includes the variables that were recorded during the experiment. The experimental variables are saved as a collection of numpy arrays. Each numpy array contains the value of that variable at a specific image presentation (i.e. trial). Note that the name of the files does not contain any information about the order or time at which the trials took place in experimental time. They are randomly ordered.
+  - **images:** This directory contains NumPy arrays where each single `X.npy` contains the image that was shown to the mouse in trial `X`.
+  - **responses:** This directory contains NumPy arrays where each single `X.npy` contains the deconvolved calcium traces (i.e. responses) recorded from the mouse in trial `X` in response to the particular presented image.
+  - **behavior:** Behavioral variables include pupil dilation, the derivative of the pupil size, and running speed. The directory contain NumPy arrays (of size `1 x 3`) where each single `X.npy` contains the behavioral variables (in the same order that was mentioned earlier) for trial `X`.
+  - **pupil_center:** the eye position of the mouse, estimated as the center of the pupil. The directory contain NumPy arrays (of size `1 x 2`) for horizontal and vertical eye positions.
 - **meta:** includes meta data of the experiment
     - **neurons:** This directory contains neuron-specific information. Below are a list of important variables in this directory
         - `area.npy`: contains the area of each neuron
-        - `cell_motor_coordinates.npy`: contains the position (x, y, z) of each neuron in the cortex
+        - `cell_motor_coordinates.npy`: contains the position (x, y, z) of each neuron in the cortex, given in microns. **Note:** The
```

Comment:
The text after "**Note:** The" is missing.

Reply:
fixed